README.md

```diff
@@ -70,6 +70,18 @@ Since 1.3.0:
 * `quarkus.tools.starter.showExtensionDescriptions`: Determines whether to show the Quarkus extension descriptions when selecting Quarkus extensions. Default is `true`.
 * `quarkus.tools.validation.value.severity`: Validation severity for property values for Quarkus/MicroProfile `*.properties` files. Default is `error`.
 
+### <mp-rest-client-class>/mp-rest/providers/<mp-rest-provider-class>/priority
+
+By waiting a fix for[this issue](https://github.com/redhat-developer/quarkus-ls/issues/203), the mp rest `<mp-rest-client-class>/mp-rest/providers/<mp-rest-provider-class>/priority` reports an unknown error. 
```

Comment:
"By waiting a fix for [this issue]" doesn't sound completely right.

How about "As we wait for a fix for [this issue]" or "Due to [this issue]"?